### PR TITLE
Updated README with cron job info

### PR DESCRIPTION
**Pull request by meenubhooshanan**

### CODE REVIEW

File: imls-wifi-sensor/README.md [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
 
-## Running session-counter
 Assumes you are a developer with git and Golang installed
 
+## Changing the Time to Send Data
+Modify session-counter.ini in imls-wifi-sensor/cmd/linux-session-counter and/or imls-wifi-sensor/cmd/windows-session-counter
+
+- If in prod:
+  - [cron]
+    reset=0 0 ** *
```
Comment (cantsin) [inline imls-wifi-sensor/README.md:10]: I think we need to wrap the cron statements in blockquotes, otherwise the `*` gets interpreted as markdown

**Review by cantsin — CHANGES_REQUESTED** [review]

File: imls-wifi-sensor/README.md (hunk 1 above)
Comment (cantsin) [reply imls-wifi-sensor/README.md:10]: or do `reset=0 0 * * *` -- either way, we're accidentally interpreting this as markdown `**` is bold, as **in bold**

Comment (meenubhooshanan) [reply imls-wifi-sensor/README.md]: Good catch, thanks!

**Review by cantsin — APPROVED** [review]
LGTM!

**Pull request merged**